esmvalcore/cmor/_fixes/shared.py

```diff
@@ -21,6 +21,19 @@ def add_scalar_height_coord(cube, height=2.0):
     return cube
 
 
+def add_scalar_height_coord_10(cube, height=10.0):
+    """Add scalar coordinate 'height' with value of `height`m."""
+    logger.debug("Adding height coordinate (%sm)", height)
+    height_coord = iris.coords.AuxCoord(height,
+                                        var_name='height',
+                                        standard_name='height',
+                                        long_name='height',
+                                        units=Unit('m'),
+                                        attributes={'positive': 'up'})
+    cube.add_aux_coord(height_coord, ())
+    return cube
+
+
```

Comment:
There is no need for this function. Just call the normal `add_scalar_height_coord` function and pass `10.0` as the second argument.